core/chains/evm/txmgr/txmgr.go

```diff
@@ -424,6 +440,18 @@ func signedTxHash(signedTx *gethTypes.Transaction, chainType chains.ChainType) (
 	return hash, nil
 }
 
+func (b *Txm) MakeForwardedTransaction(tx NewTx) (NewTx, error) {
+	fwds, count, err := b.forwardersMgr.ORM.FindForwardersByChain(utils.Big(b.chainID))
```

Comment:
In our meeting with Steve, he said we should call the tx.ToAddress, and find the list of forwarder addresses it currently allows. And then pick one from that list.
Is that going to be addressed later?